app/store/services/helpers/gps.js

```diff
@@ -0,0 +1,51 @@
+const getDistance = (
+  pointA = { longitude: 0, latitude: 0 },
+  pointB = { longitude: 0, latitude: 0 }
+) => {
+  const R = 6371e3; // metres
+  const φ1 = pointA.latitude * Math.PI / 180;
+  const φ2 = pointB.latitude * Math.PI / 180;
+  const Δφ = (pointB.latitude - pointA.latitude) * Math.PI / 180;
+  const Δλ = (pointB.longitude - pointA.longitude) * Math.PI / 180;
+
+  const a =
+    Math.sin(Δφ / 2) * Math.sin(Δφ / 2) +
+    Math.cos(φ1) * Math.cos(φ2) * Math.sin(Δλ / 2) * Math.sin(Δλ / 2);
+  const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a));
+
+  return R * c / 1000;
+};
+
+const isInArea = (
```

Comment:
is that usefull?

Reply:
Soon! to check if athlete current position is correct!